bricksrc/sensor.py

```diff
@@ -255,379 +266,469 @@
                 }
             },
             "Frequency_Sensor": {
-                "substances": [ [ BRICK.measures, BRICK.Frequency ], ],
-                "tags": [ TAG.Sensor, TAG.Frequency ],
+                "substances": [[BRICK.measures, BRICK.Frequency]],
+                "tags": [TAG.Sensor, TAG.Frequency],
                 "subclasses": {
                     "Output_Frequency_Sensor": {
-                        "tags": [ TAG.Output, TAG.Frequency, TAG.Sensor ],
+                        "tags": [TAG.Output, TAG.Frequency, TAG.Sensor],
                     }
                 }
             },
             "Frost_Sensor": {
-                "tags": [ TAG.Sensor, TAG.Frost ],
-                "substances": [ [ BRICK.measures, BRICK.Frost ], ],
+                "tags": [TAG.Sensor, TAG.Frost],
+                "substances": [[BRICK.measures, BRICK.Frost]],
+            },
+            "Gas_Sensor": {
+                SKOS.definition: Literal("senses or detects gas concentration (other than CO2)"),
+                "tags": [TAG.Sensor, TAG.Gas],
             },
             "Hail_Sensor": {
-                "tags": [ TAG.Sensor, TAG.Hail ],
-                "substances": [ [ BRICK.measures, BRICK.Hail ], ],
+                "tags": [TAG.Sensor, TAG.Hail],
+                "substances": [[BRICK.measures, BRICK.Hail]],
+            },
+            "Heat_Sensor": {
```

Comment:
What's heat sensor? heating power? thermal energy? temperature?

Reply:
Let's leave it as it is for now as it is used in practice though we don't have a clear def.